.github/workflows/pr.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v2
       with:
-        go-version: 1.19
+        go-version: 1.20.3
```

Comment:
Don't need to specify the minor part for the version, we'll update it before next releasing.

Reply:
Thanks. I will update the code.
This is due to `1.20` being treated as `1.2` in `actions/setup-go@2`.
I found a way to fix it but forgot to update this PR.